packages/frontend/apps/web/src/nbstore.ts

```diff
@@ -19,28 +19,14 @@ const consumer = new WorkerConsumer([
 
 if ('onconnect' in globalThis) {
   // if in shared worker
-  let activeConnectionCount = 0;
 
   (globalThis as any).onconnect = (event: MessageEvent) => {
-    activeConnectionCount++;
     const port = event.ports[0];
-    port.addEventListener('message', (event: MessageEvent) => {
-      if (event.data.type === '__close__') {
-        activeConnectionCount--;
-        if (activeConnectionCount === 0) {
-          globalThis.close();
```

Comment:
when will sharedworker closed now

Reply:
If all tabs are closed, the browser will automatically close it